crates/utils/re_tuid/src/lib.rs

```diff
@@ -11,15 +11,18 @@
 /// Time-ordered globally unique 128-bit identifiers.
 #[repr(C, align(1))]
 #[derive(Clone, Copy, PartialEq, Eq, Hash)]
-#[cfg_attr(feature = "bytemuck", derive(bytemuck::AnyBitPattern))]
+#[cfg_attr(
+    feature = "bytemuck",
+    derive(bytemuck::AnyBitPattern, bytemuck::NoUninit)
+)]
 pub struct Tuid {
     /// Approximate nanoseconds since epoch.
-    /// A LE u64 encoded as bytes to keep the alignment of `Tuid` to 1.
+    /// A BE u64 encoded as bytes to keep the alignment of `Tuid` to 1.
```

Comment:
A comment explaining the (unsual) choice of big-endianess (+ link to the arrow spec) would be nice